server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestFileDescriptor.java

```diff
@@ -205,10 +204,10 @@ void test() {
   }
 
   private void assertHover(AnalysisResult result) {
-    UriDecodeService uriDecodeService = mock(UriDecodeService.class);
-    when(uriDecodeService.decode(DOCUMENT_URI)).thenReturn(DOCUMENT_URI);
+//    UriDecodeService uriDecodeService = mock(UriDecodeService.class);
```

Comment:
Do we need these lines?

Reply:
Agree.
removed commented out code at e1e6fdb58aa6bea9f9708c9d1a505c81d544b93d